Tasks/SqlAzureDacpacDeploymentV1/Utility.ps1

```diff
@@ -26,86 +26,14 @@ function Get-FormattedSqlUsername
             $sqlServerFirstName = $serverNameSplittedArgs[0]
             if ((-not $sqlUsername.Trim().Contains("@" + $sqlServerFirstName)) -and $sqlUsername.Contains('@'))
             {
-                $sqlUsername = $sqlUsername + "@" + $serverName 
+                $sqlUsername = $sqlUsername + "@" + $serverName
             }
         }
     }
 
     return $sqlUsername
 }
 
-function Get-AgentIPRange
```

Comment:
Any reason to move this out of utility?

Reply:
Earlier this method has its own logic for `invoke-sqlcmd`, either we have to handle all authentication mechanisms here too or can use the already defined method `Run-InlineSql` in `SqlAzureActions.ps1` file.  That's why I have to move this method out of the utility.ps1